geode-old-versions/build.gradle

```diff
@@ -65,6 +92,21 @@ task createGeodeClasspathsFile  {
     new FileOutputStream(classpathsFile).withStream { fos ->
       versions.store(fos, '')
     }
+
+    installsFile.getParentFile().mkdirs();
+
+    new FileOutputStream(installsFile).withStream { fos ->
+      project.ext.installs.store(fos, '')
+    }
   }
+
+  // Add sourceSets for backwards compatibility, rolling upgrade, and
+  // pdx testing.
+  addOldVersion('test100', '1.0.0-incubating')
+  addOldVersion('test110', '1.1.0')
+  addOldVersion('test111', '1.1.1')
```

Comment:
Good idea, I'll add a boolean to decide whether to do the product install or not